infra/base-images/base-runner/run_fuzzer

```diff
@@ -132,6 +132,10 @@ if [[ "$FUZZING_ENGINE" = afl ]]; then
 
   CMD_LINE="$OUT/afl-fuzz $AFL_FUZZER_ARGS -i $CORPUS_DIR -o $FUZZER_OUT $(get_dictionary) $* -- $OUT/$FUZZER"
 
+  if [[ "$FUZZER" == *"non_persistent"* ]]; then
```

Comment:
Maybe put a TODO here to reference an issue for a long term solution to identifying non-persistent AFL fuzz targets.

@vanhauser-thc Do you have an ideas about how we can identify if a fuzz target uses persistent mode or not?
We can't just look for `__afl_manual_init` can we?
Note that LLVMFuzzerTestOneInput will also be in the binary so that the rest of the infra can identify it as a fuzz target.

Reply:
Thanks, can you change this to `TODO(https://github.com/google/oss-fuzz/issues/7347):...`?

Reply:
the AFL way is to check for the string `##SIG_AFL_PERSISTENT##` in the binary. if that is present, then it is persistent.
for deferred forkserver it is `##SIG_AFL_DEFER_FORKSRV##`
note that you can have the forkserver (well you should have one, better performance) without persistent mode - and vice versa.

Reply:
> note that you can have the forkserver (well you should have one, better performance) without persistent mode - and vice versa.

Interesting -- is this by setting N here to 1 https://github.com/AFLplusplus/AFLplusplus/blob/1d4f1e48797c064ee71441ba555b29fc3f467983/utils/aflpp_driver/aflpp_driver.c#L336 using this logic https://github.com/AFLplusplus/AFLplusplus/blob/1d4f1e48797c064ee71441ba555b29fc3f467983/utils/aflpp_driver/aflpp_driver.c#L306 ?

Reply:
no.
if you use the aflpp driver you automatically have persistent mode. (and a deferred forkserver).
for non-persistent mode you dont need to (and you should not) use aflppdriver

Reply:
Hey David can you implement a solution based on Marc's feedback.
Basically check if the binary doesn't contain `##SIG_AFL_PERSISTENT##` and then treat it as non-persistent.